taxonomies/categories.txt

```diff
@@ -22544,40 +22545,47 @@ fr:Crèmes Chantilly, Crème Chantilly, Chantilly
 de:Crème Chantilly
 wikidata:en:Q6384329
 
+<en:Creams
+en:UHT Creams
+fr:Crèmes UHT
+
 <en:Creams
 fr:Crèmes entières, Crème entière
 nl:Volle slagromen
 
-<fr:Crèmes entières
+<en:Creams
+fr:Crèmes liquides, crèmes fraîches liquides, Crème fraîche liquide, Crème liquide
+
+<en:Creams
+fr:Crèmes légères, crèmes fraîches légères, Crème fraîche légère, crème légère
+
+<en:Creams
+fr:Crèmes allégées en matière grasse, Crèmes fraîches allégées en matière grasse, Crème fraîche allégée, Crèmes fraîches allégées, Crème fraîche allegée en matière grasse
+nl:Light zure romen
+
+<fr:Crèmes fraîches
 <en:Sour creams
+fr:Crèmes épaisses, crèmes fraîches épaisses, Crème fraîche épaisse
+
+<en:Creams
+fr:Crèmes fraîches
+
+<fr:Crèmes épaisses
+<fr:Crèmes entières
 fr:Crème Fraîche d'Isigny
 
 <en:Creams
 <en:Fermented milk products
 en:Sour creams, Crèmes fraîches
 de:Sauerrahm
-fr:Crèmes fraîches, Crème fraîche
+fr:Crèmes fermentées, crèmes aigres
```

Comment:
Crème fraiche is a synonym of Crème fermentée. The creme fraiches contain the ingredient lactic ferments, just for that.

Reply:
If I understand well what I found on Wikipédia and in the regulation, Crème fraîche as Nothing to do with fermentation. A cream can be fresh and not fermented like this one https://world.openfoodfacts.org/product/5400247029218/creme-fraiche-fluide-winny or fermented and not fresh like this one https://world.openfoodfacts.org/product/3268630203155/creme-epaisse-val-de-retz . "Fraîche" seems to be just an indicator of the "freshness" of the cream.

Reply:
French regulation about "Crème fraîche" is : 

> Peut être mise en vente sous la dénomination de "crème fraîche" ou "crème légère fraîche", dans un délai maximum fixé par arrêté du ministre de l'agriculture, la crème ou la crème légère satisfaisant aux conditions suivantes :
> Ne pas avoir subi un traitement thermique d'assainissement autre que celui de pasteurisation ;
> Avoir été conditionnée sur le lieu de production dans les vingt-quatre heures suivant celle-ci.

Translated in English it is approximatly : "May be sold as "crème fraîche" […] cream or light cream which hasen't passed any other heat treatment than pasteurization, which has been packaged on the location of production under 24h". https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=LEGITEXT000006063234&dateTexte=20190714